samples/bot-message-reminder/nodejs/appPackage/manifest.json

```diff
@@ -0,0 +1,56 @@
+﻿{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.11/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.11",
+  "version": "1.0.0",
+  "id": "{{Microsoft-App-Id}}",
+  "packageName": "com.microsoft.teams.messagingextensionmessagereminder",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://www.microsoft.com",
+    "privacyUrl": "https://www.microsoft.com/privacy",
+    "termsOfUseUrl": "https://www.microsoft.com/termsofuse"
+  },
+  "name": {
+    "short": "Message reminder from ME",
+    "full": "Message reminder from ME"
+  },
+  "description": {
+    "short": "Message reminder from ME",
+    "full": "This sample shows feature where user can schedule a task from messaging extension action and get a reminder card at scheduled date and time."
```

Comment:
This sample shows a feature where user can schedule a task from messaging extension action and get a reminder card at a scheduled date and time.